packages/wrangler/src/pages/build-env.ts

```diff
@@ -34,24 +38,30 @@ export const Handler = async (args: PagesBuildEnvArgs) => {
 	}
 
 	const configPath = path.resolve(args.projectDir, "wrangler.toml");
-	// Fail early if the config file doesn't exist
 	if (!existsSync(configPath)) {
-		throw new FatalError(
-			"No Pages config file found",
-			EXIT_CODE_NO_CONFIG_FOUND
-		);
+		logger.log("No Pages config file found. Skipping...");
+		process.exit(EXIT_CODE_NO_CONFIG_FOUND);
```

Comment:
Let's not actually call `process.exit()` deep in the codebase. We should only do this at the top level. 

There are a couple of other alternatives: 

1. Throw a well defined typed error that can be caught at the very top level. Perhaps even a `FatalError` that has a "no log" flag. But preferably a error something like `ExitWithCodeError`.
2. Set the exit code using `process. exitCode = ...` and return here, which will then allow the main to exit with that code.